src/layers/legacy/medVtkDataMeshBase/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ target_include_directories(${TARGET_NAME}
 
 target_link_libraries(${TARGET_NAME}
   ${QT_LIBRARIES}
+  ${ITK_LIBRARIES}
```

Comment:
In medInria3, isn't it empty?

Reply:
ITK's `find_package` fills this variable with correct librairies needed to link, depending on which components were requested.